lib/src/client.rs

```
@@ -393,7 +405,10 @@ impl ClientInner {
                         Arc::clone(&network),
                         config.zkp.prover_active,
                         None,
-                        config.zkp.prover_keys_path,
+                        config
+                            .zkp
+                            .prover_keys_path
+                            .expect("Prover node must have a path for the zkp keys."),
```

Isn't it better to return an error?